docs/talks.rst

```diff
@@ -1,5 +1,23 @@
+Talks
+=====
+
+We are always interested in talking about :code:`pyhf`. See the abstract and a list of previously given presentations and feel free to invite us to your next conference/workshop/meeting!
+
+Abstract
+--------
+
+    The HistFactory p.d.f. template [CERN-OPEN-2012-016] is per-se independent of
```

Comment:
@kratsg What do you think about having the papers we cite here be hyperlinked? So [[CERN-OPEN-2012-016](https://cds.cern.ch/record/1456844)] and [[arxiv:1007.1727](https://arxiv.org/abs/1007.1727)]?